### DISCUSSION TITLE
CoastalME coding standards

### DISCUSSION THREAD

**Discussion by davefavismortlock** [coastalme/coastalme#5] [Ideas]
Hi everyone -- when changing CoastalME code, would it be useful if whoever is making the changes adds their initials to some (all?) of their intra-code comments? Maybe with a date too? 

So for example, here's a comment from Andres:

// APayo March 2018 replaced m_dMinSWL by m_dThisIterSWL

Or would this be too much like micro-management?

Dave

**Comment by wilfchun** [comment]
Hi Dave, this shouldn't really be required provided we use github correctly.
Every change done, once committed will be linked with the authors github profile.
This means that, we can use the git blame to see who has mad any change.

**Reply by davefavismortlock** [reply]
OK, fair enough. I probably have some deep psychological barrier against relying on GitHub too much... :-)
